config/setupMPI.cmake

```diff
@@ -378,7 +379,12 @@ macro( setupSpectrumMPI )
   #                         total
   # - jsrun -a4 -c16 -g2 => 4 tasks, 16 cores, 2 gpus
 
-  set( MPIEXEC_PREFLAGS "--threads=1 --bind=off -v")
+  if( MPIEXEC_EXECUTABLE MATCHES jsrun )
+    # 1 resource set; 1 thread; no gpu; no binding --nrs 1
+    set( MPIEXEC_PREFLAGS "-c 1 -g 0 --bind none")
+  elseif( MPIEXEC_EXECUTABLE MATCHES lrun )
```

Comment:
Is this meant to always pick `jsrun` unless it's not available?

Reply:
It will default to `jsrun`.  The developer can run cmake as `cmake -DMPIEXEC_EXECUTABLE=lrun` to override.  LLNL keeps telling me that I need to use `lrun`, but `lrun` fails badly for me. I'm trying to keep the build system compatible with both launchers.

Reply:
OK, thanks for the clarification!